packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.js

```diff
@@ -52,6 +52,14 @@ export default class QuanticCategoryFacetValue extends LightningElement {
     return this.item.value;
   }
 
+  get ariaLabelVoiceOver() {
+    return `Inclusion filter on ${this.facetValue};`;
+  }
+
+  get isPressed() {
+    return this.item.state === 'selected';
```

Comment:
Just to be perfectly safe if something doesn't exist. Otherwise your html will have `aria-pressed="undefined"` or the code could throw an error.
```suggestion
    return this.item?.state === 'selected';
```

Reply:
:white_check_mark: